ansible/roles/k3s/tasks/agent-runtime.yml

```diff
@@ -0,0 +1,32 @@
+---
+
+- name: Template k3s agent env file
```

Comment:
We should set owner/group/mode restrictively here so normal users cannot access the token